packages/dscc-gen/src/util.ts

```diff
@@ -24,50 +23,25 @@ import * as fs from 'fs';
 import {Answers} from 'inquirer';
 
 export const readDir = (path: string): Promise<string[]> => {
```

Comment:
If we want this to return a promise still, we can just make it async and get rid of the Promise constructor boiler plate

Reply:
k

Reply:
I think I want to make this synchronous like everything else.